common/src/main/resources/data/vs_tournament/vs_mass/tournament.json

```diff
@@ -26,5 +26,9 @@
   {
     "block": "vs_tournament:sensor",
     "mass": 200.0
+  },
+  {
+    "block": "vs_tournament:floater",
+    "mass": 50.0
```

Comment:
A bit too much buoyancy

oak planks have a weight of 600 (~ 1.6 times lighter than water)

This is 20 times lighter

Reply:
Do you think 200 kg is fine? Currently Ballast has a base weight lower than that so idk exactly what would be balanced per say.

Reply:
Ah yeah exactly
forgot about that
we should make the ballast base weight a bit higher.
People have been using the ballast as floater.
The ballast base should be about 1 times as heavy as water (=1000kg), so it doesn't have buyoancy.
Also we should make sure that the ballast recipe makes sense.

200kg for the floater doesn't seem too bad. But we should have people test it with 200kg and they should give us an honest feedback if it is too much or not enought buyoancy